relay_test.go

```diff
@@ -565,6 +565,12 @@ func TestRelayRejectsDuringClose(t *testing.T) {
 		require.Error(t, err, "Expect call to fail after relay is shutdown")
 		assert.Contains(t, err.Error(), "incoming connection is not active")
 		close(block)
+		wg.Wait()
+		// FIXME:
+		// We fire all pending timers when we shut down the relay, which
+		// includes the timeout for the blocked handler. That unblocks the
+		// handler before we get to the call above, which obviates the point of
+		// this test.
```

Comment:
We should consider whether we really want to fire all timers when stopping the wheel, since that times out all in-flight calls.